packages/internal-lib-build/typedoc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "out": "../../docs",
+  "tsconfig": "../../tsconfig.json",
+  "entryPoints": [
+    "../pwa-kit-react-sdk/src/index.ts",
```

Comment:
nice! so we can use relative paths to specify the packages

Reply:
Apparently, it can, but currently, the tsconfig file has to be at root, otherwise, Typedoc could not recognize other entry point, which makes me wondering if this might be the best approach.